### PR TITLE
"Adding Calendar Text Section"

### PR DESCRIPTION
**Pull request by DeneLogan**

### CODE REVIEW

**Comment by DeneLogan** [comment]
I was having issues figuring out how to add the links to this page, so I left it as a To-Do.

File: data/calendar.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+enable : true
+calendar_notes  :
```
Comment (bluelotus03) [inline data/calendar.yml:2]: For simplicity of upkeep, I think the "Virtual" and "In-person" sections can be set up the same way you have the "Main Events one, where they are their own sections.

**Review by bluelotus03 — CHANGES_REQUESTED** [review]
Looking great! 
We will talk a bit more about linking text today to help with finalizing this issue. 

Then you can continue making changes and pushing them to your branch, and it will automatically update the PR here. So you won't have to make a new one. That way you can comment and tag me with "@bluelotus03" to let me know when you're done making updates.

If you have any questions, just lemme know. Awesome job!

File: data/calendar.yml (hunk 1 above)
Comment (DeneLogan) [reply data/calendar.yml]: @bluelotus03 All done :)!

Comment (bluelotus03) [reply data/calendar.yml]: Awesome job!

**Review by bluelotus03 — APPROVED** [review]
Looks great!

**Pull request merged**